src/databricks/labs/ucx/hive_metastore/grants.py

```diff
@@ -109,6 +110,15 @@ def object_key(self) -> str:
         _, key = self.this_type_and_key()
         return key.lower()
 
+    @property
+    def order(self) -> int:
+        """Order of the grants to be upheld when applying."""
+        match self.action_type:
+            case "OWN":
+                return 1
+            case _:
+                return 0
```

Comment:
Because I like to be fancy and show off 💁‍♂️ 

It is copied from: https://github.com/databrickslabs/ucx/blob/d3d08f9c3de178258150b0ea3fcd93cb4aedb836/src/databricks/labs/ucx/installer/workflows.py#L766-L776. 

Though, with just two values `0` and `1` it maybe does not matter